e2e/e2etests/e2etests.go

```diff
@@ -280,6 +282,16 @@ var AllE2ETests = []runner.E2ETest{
 	/*
 	 EVM chain tests
 	*/
+	// TestV2ZetaDepositName is not used for now as the protocol logic has not yet been implemented
+	// // https://github.com/zeta-chain/node/issues/3212
+	runner.NewE2ETest(
+		TestV2ZetaDepositName,
```

Comment:
I'm not sure we need the v2 naming, old tests should already be marked as legacy
These will be the default regular ZETA tests